Source/Web/app/HeatMap.js

```diff
@@ -1,33 +1,43 @@
+/*
+*	File:	HeatMap.js
+*	Authour:	Binary Ninjaz (Letanyan, John, Vincent)
+*
+*	Description: 	This file requests and receives data from firebase database
+*					To display a heat map on >HeatMap.html". It shows heat where 
+*					bags were collected.
+*/
+/* This function returns the authentication details of the current user from the database*/
 const user = function() { return firebase.auth().currentUser };
+/* This function checks whether the user shoukd have access or not*/
 const userID = function() {
   if (user() !== null) {
     return user().uid 
   } else {
     return ""
   }
 }
-
+/* This function gets a "snapshot" of the ochards (given the particular userID) from firebase*/
```

Comment:
This functions gets all the orchards and executes the callback with the snapshot as the parameter.